src/components/atoms/index.ts

```diff
@@ -1,3 +1,2 @@
-export * from './Logo';
```

Comment:
Why did u remove the logo component?

Reply:
because it added no value

Reply:
Hmmm I don't know really but It's better to ask @mojtabaesl first. because the logo is a part of atoms design.